### ISSUE TITLE
Linking failed with React Native. infobip-mobile-messaging-react-native-plugin.podspec file is not installing

### ISSUE THREAD

**Issue by pratius** [infobip/mobile-messaging-react-native-plugin#1]
Hi,

We are trying to integrate this Plugin to the React Native project. but always throwing the attached exception. Below code to initialize push notification with application code.
```
const { ReactNativeMobileMessaging } = NativeModules;

    ReactNativeMobileMessaging.init("3c59f6e341a6896fc05b8cd7e3f3fdf8-031a75db-fd8f-46b0-9f2b-a2e915d7b952_");
     
```

Can you please share the POC/Example for React Native app? 
And also, Let's know how to link with React Native.

Thanks,
![Simulator Screen Shot - iPhone 11 - 2020-02-20 at 14 15 06](https://user-images.githubusercontent.com/15229476/74924447-f02b2200-53eb-11ea-93aa-da9df56c60a9.png)

**Comment by nsustavov** [comment]
Hi @pratius 

As it is said in [readme](https://github.com/infobip/mobile-messaging-react-native-plugin/blob/master/README.md) Infobip ReactNative plugin is still in development. 

Our approx date of release mid of March. We will notify you in this thread on public release.

**Comment by tjuric** [comment]
Hi @pratius,
As you might've already seen, yesterday we've released initial version [1.0.0](https://github.com/infobip/mobile-messaging-react-native-plugin/releases/tag/1.0.0) of our React Native plugin. 
Quick start guide might be found in [readme](https://github.com/infobip/mobile-messaging-react-native-plugin/blob/master/README.md) file and other guidelines on the usage can be found on project's [Wiki pages](https://github.com/infobip/mobile-messaging-react-native-plugin/wiki).

In case you encounter some issue with plugin integration you can open the issue here or contact our support on push.support@infobip.com.

BR,
Tereza

**Comment by pratius** [comment]
Hi Tereza,

Thank you for RN Plugin. 

I have implemented the below code to insert user information on the Infobip portal. But always inserting unexpected information on Portal and Push notification is not working. 

 **Requirement** 
 We want to register an email address on the portal. Because as per our requirements sending to push notification on phone, we are using user email Id and push ID.
 But in React Native case ID parameter is always inserting default random ID. 

Maybe because of that, failing the Notification in React Native. 

**Note:-**I have done the same implementation in native iOS it's inserting expected information on Portal and working notification also for the phone.

 I have attached both images for references. 

**Implementation:-**

**Calling for initialization:-**
`import { mobileMessaging } from "infobip-mobile-messaging-react-native-plugin";`
  mobileMessaging.init(
      {```

        applicationCode:
          "here infobip application code",
        ios: {
          notificationTypes: ["alert", "badge", "sound"]
        }
      },
      () => {
        console.log('MobileMessaging started');
      },
      error => {
        console.log('MobileMessaging error: ', error);
      },
    );

    mobileMessaging.register('registrationUpdated', message => {
      console.log('Event received', message);

    });

```

**User information Register to Infobip Portal:-**

```
const user = {
      externalUserId: "testadmin@gmail.com",
      firstName: "PP",
      lastName: "Dubey",
      phones: ["9710000000"],
      emails: ["testadmin@gmail.com"],
    };
    );

    mobileMessaging.saveUser(
      user,
      user => {
        console.log("Inserted data on Success", user);
      },
      user => {
        console.log("Inserted data error", user);
      }
    );
```
Please help us with this scenario and let's know if we are missing any implementation for the same. 

Thanks,
Pratius

<img width="1211" alt="iOS-Native-Portal-ScreenShot" src="https://user-images.githubusercontent.com/15229476/76854560-bde4d700-6868-11ea-8dd8-32dc5588e803.png">
<img width="1216" alt="React-Native-Portal-Screen-Shots" src="https://user-images.githubusercontent.com/15229476/76854566-c0dfc780-6868-11ea-85a3-9fea32015c85.png">
